test/UnbrandedProjects/Platform-OpenAI-TypeSpec/src/Generated/Models/CreateImageEditRequest.Serialization.cs

```diff
@@ -175,6 +190,52 @@ internal static CreateImageEditRequest DeserializeCreateImageEditRequest(JsonEle
                 serializedAdditionalRawData);
         }
 
+        private BinaryData SerializeMultipart(ModelReaderWriterOptions options)
+        {
+            using MultipartFormDataBinaryContent content = ToMultipartBinaryBody();
+            using MemoryStream stream = new MemoryStream();
+            content.WriteTo(stream);
+            if (stream.Position > int.MaxValue)
+            {
+                return BinaryData.FromStream(stream);
+            }
+            else
+            {
+                return new BinaryData(stream.GetBuffer().AsMemory(0, (int)stream.Position));
+            }
+        }
+
+        internal virtual MultipartFormDataBinaryContent ToMultipartBinaryBody()
+        {
+            MultipartFormDataBinaryContent content = new MultipartFormDataBinaryContent();
+            content.Add(Prompt, "prompt");
+            content.Add(Image, "image", "image", "application/octet-stream");
+            if (Optional.IsDefined(Mask))
+            {
+                content.Add(Mask, "mask", "mask", "application/octet-stream");
```

Comment:
This will not work -- OpenAI needs to be able to pass the file name to this method to correctly format the header for the Mask property in the multipart/form-data content payload.